### PR TITLE
Add bridging of GCD from BSC in one command

### PR DESCRIPTION
**Pull request by drinkius**

### CODE REVIEW

File: Parser/Updating/Parser.ts [hunk 1]
```diff
@@ -67,82 +66,91 @@ const HelpWorker = ({ print }) => {
...
     if (tmpARGS.length < 7) {
       throw new Error('Invalid input');
     }
 
-    const Amount = tmpARGS[2];
-    const TokenName = tmpARGS[3];
-    const percentRisk = +parseInt(tmpARGS[5]) / 100;
+  const Amount = tmpARGS[2];
+  const TokenName = tmpARGS[3];
+  const percentRisk = +parseInt(tmpARGS[5]) / 100;
```
Comment (slavastartsev) [inline Parser/Updating/Parser.ts:76]: We can get rid of either `+` or `parseInt()`. The only difference is that `parseInt('1234abc') -> 1234` while +`'1234abc' -> NaN` (so `+` is essentially identical to calling `Number()` on an argument)

File: Parser/WEB3/chains.ts [hunk 2]
```diff
@@ -26,11 +26,11 @@ export const mmChains: MMChains = {
...
   },
   gton: {
-    chainId: '50021',
-    chainIdHex: '0xC365',
-    chainName: 'TestGTON',
-    rpcUrls: ['https://testnet.gton.network'],
+    chainId: '1000',
+    chainIdHex: '0x3E8',
+    chainName: 'GTON',
+    rpcUrls: ['https://gton.network'],
```
Comment (slavastartsev) [inline Parser/WEB3/chains.ts:32]:
```suggestion
    rpcUrls: ['https://rpc.gton.network'],
```

File: Parser/WEB3/chains.ts [hunk 3]
```diff
@@ -26,11 +26,11 @@ export const mmChains: MMChains = {
...
     nativeCurrency: { name: 'GCD', decimals: 18, symbol: 'GCD' },
-    blockExplorerUrls: ['https://explorer.testnet.gton.network/'],
+    blockExplorerUrls: ['https://testnet.gton.network/'],
```
Comment (slavastartsev) [inline Parser/WEB3/chains.ts:34]:
```suggestion
    blockExplorerUrls: ['https://explorer.gton.network/'],
```

File: config/config.ts [hunk 4]
```diff
@@ -1,25 +1,32 @@
-const network = '1'; // Ethereum
+const gtonTokenNetwork = '1'; // Ethereum
+const rollupL1NetworkId = '56'; // BSC
 const isDev = process.env.NODE_ENV === 'development';
 const isLive = 'true';
 const isTestnet = false;
 const claimNetwork = '250';
+const bsc = '56';
```
Comment (slavastartsev) [inline config/config.ts:7]: Duplicate

File: config/config.ts [hunk 5]
```diff
@@ -1,25 +1,32 @@
-const network = '1'; // Ethereum
+const gtonTokenNetwork = '1'; // Ethereum
+const rollupL1NetworkId = '56'; // BSC
```
Comment (slavastartsev) [inline config/config.ts:2]: Duplicate

File: Messages/Messages.tsx [hunk 6]
```diff
@@ -14,6 +14,7 @@ enum Commands {
   CLAIM = 'claim',
   PRICE = 'price',
   SWITCH = 'switch',
+  BUYANDBRIDGEGCD = 'bridge',
```
Comment (slavastartsev) [inline Messages/Messages.tsx:17]:
```suggestion
  BUY_AND_BRIDGE_GCD = 'bridge',
```

File: components/Header/Header.tsx [hunk 7]
```diff
@@ -80,6 +80,10 @@ function Header() {
           ? 'ROPSTEN'
```
Comment (PoulWalcker) [inline components/Header/Header.tsx:80]: Ropsten is unavailable now. We need to remove it

File: config/config-test.ts [hunk 8]
```diff
@@ -3,10 +3,11 @@ const isDev = process.env.NODE_ENV === 'development';
 const isLive = 'true';
```
Comment (slavastartsev) [inline config/config-test.ts:3]: Line 1 in this file refers to chain id of Ropsten network. WDYT about changing it to 97 (BSC Testnet)?

File: Messages/Messages.tsx [hunk 9]
```diff
@@ -66,14 +67,10 @@ const cdHelp = `
...
-} gton | sgton | harvest | gcd | all - get actual erc20 token balance
-  ${Prefix.PREFIX}${Commands.ADD_TOKEN} gton | sgton | gcd | usdc  - add tokens to metamask,
-  ${Prefix.PREFIX}${
-  Commands.BUY
-} gton for <amount> <token>  - available tokens: eth, usdc (more coming soon) 
-  ${isTestnet ? `${Prefix.PREFIX}${Commands.SWITCH} - switch chain to GTON` : ''}
+  ${Prefix.PREFIX}${Commands.BALANCE} gton | sgton | harvest | gcd | all - get actual erc20 token balance
+  ${Prefix.PREFIX}${Commands.BUYANDBRIDGEGCD} <amount> <token> - buy GCD with stable at ~0.92 liquidation price and bridge it to rollup
+  ${Prefix.PREFIX}${Commands.ADD_TOKEN} gton | sgton | gcd | usdc - add tokens to metamask
+  ${Prefix.PREFIX}${Commands.BUY} gton for <amount> <token>  - available tokens: busd, usdc (more coming soon)
```
Comment (PoulWalcker) [inline Messages/Messages.tsx:73]: Do we have pool GTON/BUSD to buy for busd?

Comment (drinkius) [reply Messages/Messages.tsx:73]: No, just the stable protocol

File: components/Header/Header.tsx (hunk 7 above)
Comment (drinkius) [reply components/Header/Header.tsx:80]: Can fix in the next iteration

File: config/config.ts (hunk 4 above)
Comment (drinkius) [reply config/config.ts:7]: Also not duplicate - this is used in different places.

We need to make a thorough refactoring to optimize such places

File: Messages/Messages.tsx (hunk 6 above)
Comment (drinkius) [reply Messages/Messages.tsx:17]: Fixed

File: Parser/Updating/Parser.ts (hunk 1 above)
Comment (drinkius) [reply Parser/Updating/Parser.ts:76]: Fixed

File: Parser/Updating/Parser.ts [hunk 10]
```diff
@@ -67,82 +66,91 @@ const HelpWorker = ({ print }) => {
...
+});
+
+export async function borrowGCDWithRisk(tokenAmount, tokenName, percentRisk, userAddress, lock, loading, print): Promise<Big> {
+  const token = tokenName in collateralsTokenMap ? collateralsTokenMap[tokenName] : null;
+  if (!token) {
+    throw new Error('Wrong symbol, available tokens: busd, usdc');
+  }
+
+  if (percentRisk > 1 || percentRisk < 0.01)
+      // Converted persent
```
[the remark this answers is not in the document]
Comment (drinkius) [reply Parser/Updating/Parser.ts:88]: Fixed

File: Parser/WEB3/chains.ts (hunk 2 above)
Comment (drinkius) [reply Parser/WEB3/chains.ts:32]: Fixed

File: config/config-test.ts (hunk 8 above)
Comment (drinkius) [reply config/config-test.ts:3]: Testnet config doesn't work at all yet, should be fixed later alongside the reintroduction of the test version

**Review by slavastartsev — APPROVED** [review]
🔥

**Pull request merged**